domain/src/main/java/com/globant/domain/entity/HouseId.kt

```diff
@@ -0,0 +1,6 @@
+package com.globant.domain.entity
+
+data class HouseId(
```

Comment:
please rename this class to House, as this is not only an ID (check the related code)

Reply:
Refactored!